### PR TITLE
Rewrited lf_em4100rwc standalone mode, added card bruteforce mode.

### PR DESCRIPTION
**Pull request by monster1025**
I have created separate mode - LF_EM4100RSWB(and not changed original one), because in this one is harder to switch between slots, but easier to work with one slot (more automatic switches between modes), also I reduced 16 slots to 4.

I think this 'menu/modes' layout is more suitable when you need fast read somebody's card, go to the door and sim it/write to empty card/try to brute nearest cards.

In this standalone mode I have added card bruteforce mode.

Hope now I do it right =)

### CODE REVIEW

File: armsrc/Standalone/Makefile.hal [hunk 1]
```diff
@@ -20,6 +20,9 @@ define KNOWN_STANDALONE_DEFINITIONS
 | LF_EM4100RWC    | Read/simulate em4100 tags & clone it   |
 |                 | to T555x tags                          |
 +----------------------------------------------------------+
+| LF_EM4100RSWB   | Read/simulate/brute em4100 tags &      |
```
Comment (doegox) [inline armsrc/Standalone/Makefile.hal:23]: please respect alphabetical sorting :)

File: armsrc/Standalone/Makefile.hal [hunk 2]
```diff
@@ -55,7 +58,7 @@ define KNOWN_STANDALONE_DEFINITIONS
 +----------------------------------------------------------+
 endef
 
-STANDALONE_MODES := LF_SKELETON LF_EM4100EMUL LF_EM4100RWC LF_HIDBRUTE LF_ICEHID LF_PROXBRUTE LF_SAMYRUN
+STANDALONE_MODES := LF_SKELETON LF_EM4100EMUL LF_EM4100RWC LF_EM4100RSWB LF_HIDBRUTE LF_ICEHID LF_PROXBRUTE LF_SAMYRUN
```
Comment (doegox) [inline armsrc/Standalone/Makefile.hal:61]: please respect alphabetical sorting :)

File: armsrc/Standalone/Makefile.inc [hunk 3]
```diff
@@ -49,6 +49,10 @@ endif
 ifneq (,$(findstring WITH_STANDALONE_LF_EM4100RWC,$(APP_CFLAGS)))
     SRC_STANDALONE = lf_em4100rwc.c
 endif
+# WITH_STANDALONE_LF_EM4100RSWB
```
Comment (doegox) [inline armsrc/Standalone/Makefile.inc:52]: please respect alphabetical sorting :)

File: armsrc/Standalone/Makefile.hal (hunk 1 above)
Comment (monster1025) [reply armsrc/Standalone/Makefile.hal:20]: done

File: armsrc/Standalone/Makefile.hal (hunk 2 above)
Comment (monster1025) [reply armsrc/Standalone/Makefile.hal]: done

File: armsrc/Standalone/Makefile.inc (hunk 3 above)
Comment (monster1025) [reply armsrc/Standalone/Makefile.inc:48]: done

**Comment by doegox** [comment]
thank you :)
One last thing, please get rid of these warnings
```
../armsrc/Standalone/lf_em4100rswb.c:86:10: warning: no previous prototype for 'ReversQuads' [-Wmissing-prototypes]
 uint64_t ReversQuads(uint64_t bits) {
          ^~~~~~~~~~~
../armsrc/Standalone/lf_em4100rswb.c:94:6: warning: no previous prototype for 'FillBuff' [-Wmissing-prototypes]
 void FillBuff(uint8_t bit) {
      ^~~~~~~~
../armsrc/Standalone/lf_em4100rswb.c:101:6: warning: no previous prototype for 'ConstructEM410xEmulBuf' [-Wmissing-prototypes]
 void ConstructEM410xEmulBuf(uint64_t id) {
      ^~~~~~~~~~~~~~~~~~~~~~
../armsrc/Standalone/lf_em4100rswb.c:123:6: warning: no previous prototype for 'LED_Update' [-Wmissing-prototypes]
 void LED_Update(int mode, int slot) {
      ^~~~~~~~~~
../armsrc/Standalone/lf_em4100rswb.c:155:6: warning: no previous prototype for 'FlashLEDs' [-Wmissing-prototypes]
 void FlashLEDs(uint32_t speed, uint8_t times) {
      ^~~~~~~~~
../armsrc/Standalone/lf_em4100rswb.c:166:6: warning: no previous prototype for 'SaveIDtoFlash' [-Wmissing-prototypes]
 void SaveIDtoFlash(int addr, uint64_t id) {
      ^~~~~~~~~~~~~
../armsrc/Standalone/lf_em4100rswb.c:181:10: warning: no previous prototype for 'PackEmID' [-Wmissing-prototypes]
 uint64_t PackEmID(uint64_t original, int newCardNum) {
          ^~~~~~~~
../armsrc/Standalone/lf_em4100rswb.c:201:6: warning: no previous prototype for 'PrintFcAndCardNum' [-Wmissing-prototypes]
 void PrintFcAndCardNum(uint64_t lowData) {
      ^~~~~~~~~~~~~~~~~
../armsrc/Standalone/lf_em4100rswb.c:208:5: warning: no previous prototype for 'ButeEMTag' [-Wmissing-prototypes]
 int ButeEMTag(uint64_t originalCard, int slot) {
     ^~~~~~~~~
../armsrc/Standalone/lf_em4100rswb.c:252:5: warning: no previous prototype for 'ExecuteMode' [-Wmissing-prototypes]
 int ExecuteMode(int mode, int slot) {
     ^~~~~~~~~~~
../armsrc/Standalone/lf_em4100rswb.c:285:5: warning: no previous prototype for 'SwitchMode' [-Wmissing-prototypes]
 int SwitchMode(int mode, int slot) {
     ^~~~~~~~~~
../armsrc/Standalone/lf_em4100rswb.c: In function 'RunMod':
../armsrc/Standalone/lf_em4100rswb.c:307:9: warning: declaration of 'slots_count' shadows a global declaration [-Wshadow]
     int slots_count = 4;
         ^~~~~~~~~~~
../armsrc/Standalone/lf_em4100rswb.c:79:15: note: shadowed declaration is here
 uint8_t *bba, slots_count;
               ^~~~~~~~~~~
```

Basically for *no previous prototype* put `static` and for *shadow global decl* rename your `slots_count` var.
Thanks

**Comment by doegox** [comment]
Great, thanks !

**Comment by monster1025** [comment]
Thanks a lot for your review =)

**Pull request merged**